### DISCUSSION TITLE
Is nodelet a-must for ROS1?

### DISCUSSION THREAD

**Discussion by pigking12345** [foxglove/community#679] [Q&A]
**Description**
I am trying to deploy ros-foxglove-bridge to an embedded system with ROS 1 environment. Then I found that it is required to launch both nodelet manager and foxglove-bridge. I am wondering if the nodelet mechanism is necessary for foxglove bridge.
I have several reasons:
1. The embedded system has very limited resource. Each process will consume some resource (CPU & memory) and I want to save some resource from the nodelet manager.
2. The embedded system doesn't use nodelet mechanism before. Therefore, I guess there won't be any performance gain if foxglove-bridge uses the nodelet mechanism.

- Version: ROS 1 
- Platform: Arm, Linux.

**Comment by pigking12345** [comment]
Sorry that I don't know how to change the label of the issue. It should be "question" instead of "bug".

**Reply by amacneil** [reply]
I moved it to a discussion - we prefer this format for non-bugs. Thanks!
